net-dns/knot/knot-2.0.2.ebuild

```diff
@@ -34,8 +34,7 @@ DEPEND="${RDEPEND}
 S="${WORKDIR}/${P/_/-}"
 
 src_prepare() {
-	epatch "${FILESDIR}/${PV}-spell-enable-vars-correctly.patch"
-	epatch "${FILESDIR}/${PV}-dont-create-extra-directories.patch"
+	epatch "${FILESDIR}/2.0.2-dont-create-extra-directories.patch"
```

Comment:
Since this patch is unchanged since 2.0.0, I changed ${PV} to 2.0.2 so future version bumping will be easier